sourced/ml/mining/__main__.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import sys
+
+from modelforge import slogging
+
+from sourced.ml.mining.cmd import ArgumentDefaultsHelpFormatterNoNone
+
+
+def get_parser() -> argparse.ArgumentParser:
+    """
+    Create the cmdline argument parser.
+    """
+    parser = argparse.ArgumentParser(
+        formatter_class=ArgumentDefaultsHelpFormatterNoNone
+    )
+    slogging.add_logging_args(parser, patch=True, erase_args=False)
+
+    # Create and construct subparsers
+    subparsers = parser.add_subparsers(help="Commands", dest="command")
+
+    def add_parser(name, help_message):
+        return subparsers.add_parser(
+            name, help=help_message, formatter_class=ArgumentDefaultsHelpFormatterNoNone
+        )
+
+    return parser
+
+
+def main():
+    """
+    Create all the argparse-rs and invokes the function from set_defaults().
+
+    :return: The result of the function from set_defaults().
+    """
+    parser = get_parser()
+    args = parser.parse_args()
+    try:
+        handler = args.handler
+        for arg_name in ["command", "handler", "log_structured", "log_config"]:
```

Comment:
We should not delete anything. This is the legacy from the old times we passed `**args`

Reply:
ah oke, removing this then